lib/rucio/core/replica.py

```diff
@@ -907,6 +907,19 @@ def _list_files_wo_replicas(files_wo_replica, session):
             yield scope, name, bytes, md5, adler32
 
 
+@read_session
+def remove_empty_open_did(force_refresh=False, session=None):
+    """
+    Check if empty open dids must be removed (default Yes)
```

Comment:
I like that we put it default to "yes". But it must be very clear in release notes that the default behavior changes.